tests/Constraints/FormatTest.php

```diff
@@ -23,7 +23,8 @@ public function testNullThing()
         $validator = new FormatConstraint();
         $schema = new \stdClass;
 
-        $validator->check('10', $schema);
+        $checkValue = 10;
+        $validator->check($checkValue, $schema);
```

Comment:
Just curious; is it still necessary to pass in a variable now that top-level check no longer takes a reference?